src/css/media-queries.scss

```diff
@@ -0,0 +1,16 @@
+/* media-queries*/
+@media (min-width: 640px) {
```

Comment:
I appreciate you putting in the leg work on this. I still have to have you walk me through making the queries work